client/components/domains/domain-product-price/index.jsx

```diff
@@ -25,63 +25,56 @@ class DomainProductPrice extends React.Component {
 		domainsWithPlansOnly: PropTypes.bool.isRequired,
 	};
 
-	renderFreeWithPlan() {
-		return (
-			<div
-				className={ classnames( 'domain-product-price', 'is-free-domain', {
-					'no-price': this.props.domainsWithPlansOnly,
-				} ) }
-			>
-				{ ! this.props.domainsWithPlansOnly && this.renderFreeWithPlanPrice() }
-				<span className="domain-product-price__free-text">
-					{ this.props.translate( 'Free with your plan' ) }
-				</span>
-			</div>
-		);
+	renderFreeWithPlanText() {
+		const { translate } = this.props;
+
+		let message;
+		switch ( this.props.rule ) {
+			case 'FREE_WITH_PLAN':
+				message = this.props.translate( 'First year free with your plan' );
+				break;
+			case 'INCLUDED_IN_HIGHER_PLAN':
+				message = translate( 'Included in paid plans' );
+				break;
+			case 'UPGRADE_TO_HIGHER_PLAN_TO_BUY':
+				message = translate( 'Personal plan required' );
+				break;
+		}
+
+		return <div className="domain-product-price__free-text">{ message }</div>;
 	}
 
 	renderFreeWithPlanPrice() {
 		return (
-			<span className="domain-product-price__price">
-				{ this.props.translate( '%(cost)s {{small}}/year{{/small}}', {
+			<div className="domain-product-price__price">
+				{ this.props.translate( 'Renewal: %(cost)s {{small}}/year{{/small}}', {
```

Comment:
:disappointed: 36 existing translations will be lost with this change.
If the string must change, the following option has already been translated **1** time:
`translate( '{{small}}+%(cost)s / year{{/small}}' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).